config.js

```diff
@@ -46,14 +46,20 @@ var config = {
     testing: {
         // P2P test mode disables automatic switching to P2P when there are 2
         // participants in the conference.
-        p2pTestMode: false
+        p2pTestMode: false,
 
         // Enables the test specific features consumed by jitsi-meet-torture
         // testMode: false
 
         // Disables the auto-play behavior of *all* newly created video element.
         // This is useful when the client runs on a host with limited resources.
         // noAutoPlayVideo: false
+
+        // Enable / disable 500 Kbps bitrate cap on desktop tracks. When enabled,
+        // simulcast is turned off for the desktop share. If presenter is turned
+        // on while screensharing is in progress, the max bitrate is automatically
+        // adjusted to 2.5 Mbps.
+        capScreenshareBitrate: 1 // 0 to disable
```

Comment:
Please use a Boolean instead of an integer and leave it commented out, with the default value set.

Reply:
Also, is is whitelisted already?

Reply:
If the code still uses it as a probability we should document it IMHO

Reply:
ok sure, will do that. Also its under testing which is whitelisted, we don't need to whitelist this separately right ?

Reply:
Correct!